changelog/fix-7241-update-live-dispute-docs-links-in-transaction-detail

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Update documentation links (new/changed docs content) when notifying merchant that a dispute needs response.
```

Comment:
Should this be a "comment" changelog entry since it's behind a feature flag and not visible to users?

```suggestion
Comment: Behind a feature flag: Update documentation links (new/changed docs content) when notifying merchant that a dispute needs response.
```

Reply:
Good point 😄 7dc77b0fb3fb551d93d26181f098b2c9977196db